app/about_us/OurCrew.js

```diff
@@ -0,0 +1,26 @@
+import styles from "./OurCrew.module.css";
+import { ourCrewAbout, ourCrewData } from "./crew.js";
+
+export const OurCrew = () => {
+  return (
+    <section className={styles.container}>
+      <header className={styles.crewHeader}>
+        <p>{ourCrewAbout.description}</p>
+      </header>
+      <div className={styles.crewAbout}>
+        {ourCrewData.map((member, index) => (
```

Comment:
Great job using the `.map()` function, this wasn't even part of this weeks HW 🌟

Reply:
Thank you! Is it better to use the index or add the id to each object in the array?